xmodule/capa_block.py

```diff
@@ -1753,7 +1813,10 @@ def submit_problem(self, data, override_time=False):
             self.attempts = self.attempts + 1
             self.lcp.done = True
             self.set_state_from_lcp()
-            self.set_score(self.score_from_lcp(self.lcp))
+            if self.enable_grading_method:
+                self._set_score()
+            else:
+                self.set_score(self.score_from_lcp(self.lcp))
```

Comment:
should we change the name of `_set_score` so it's different than set_score? Something that indicates the difference.